doc/pages/architecture/030-interface.md

```diff
@@ -227,6 +227,10 @@ the default for each mapping, or at mapping level, overriding any interface-wide
 * `expiry`: Meaningful only when `retention` is `stored`. Defines how many seconds a specific data
   entry should be kept before giving up and erasing it from the persistent cache. A value <= 0 means
   the persistent cache never expires, and is the default.
+* `database_retention_policy`: Useful only with datastream. Defines whether data is expired from
+  the database after a given time to live interval.
```

Comment:
`Defines whether data should expire from the database after a given interval. Valid values are: [...]`